SPTAG.nuspec

```diff
@@ -78,6 +78,9 @@
     <file src="AnnService\inc\Helper\StringConvert.h" target="build\native\inc\Helper\StringConvert.h" />
     <file src="AnnService\inc\Helper\ThreadPool.h" target="build\native\inc\Helper\ThreadPool.h" />
     <file src="AnnService\inc\Helper\VectorSetReader.h" target="build\native\inc\Helper\VectorSetReader.h" />
+    <file src="ThirdParty\zstd\lib\zdict.h" target="build\native\zdict.h" />
+    <file src="ThirdParty\zstd\lib\zstd.h" target="build\native\zstd.h" />
+    <file src="ThirdParty\zstd\lib\zstd_errors.h" target="build\native\zstd_errors.h" />
     <file src="x64\Debug\CoreLibrary.lib" target="lib\native\debug\x64\CoreLibrary.lib" />
     <file src="x64\Debug\CoreLibrary.pdb" target="lib\native\debug\x64\CoreLibrary.pdb" />
     <file src="x64\Release\CoreLibrary.lib" target="lib\native\retail\x64\CoreLibrary.lib" />
```

Comment:
You will also need to pack the libzstd.lib and libzstd.dll into the nuget package.

Reply:
We have internal zstd build, I will check if it is usable instead of packaging different version along w/ sptag